sound/soc/sof/core.c

```diff
@@ -29,13 +29,18 @@ struct snd_sof_pcm *snd_sof_find_spcm_name(struct snd_sof_dev *sdev,
 	struct snd_sof_pcm *spcm = NULL;
 
 	list_for_each_entry(spcm, &sdev->pcm_list, list) {
+		/* match with PCM dai name */
```

Comment:
can you clarify in the commit message when this can happen or what problem is fixed here. It's hard to figure out in what cases the string name would be empty and what side effects result from that confusion.